src/components/core/pivotTable/agGridApiWrapper.ts

```diff
@@ -5,11 +5,96 @@ function getHeaderHeight(gridApi: GridApi): number {
     return (gridApi as any).headerRootComp.eHeaderContainer.clientHeight;
 }
 
```

Comment:
I think that api wrapper should have just functions which grab data from unsafe agGridApi. I think it's also becoming utility lib and those utility functions should be moved to agGridHelper or somewhere else. Wrapper should be easily testable and not bloated.

Reply:
I understand your point but the point here is to cover functionality that is not supported by the official API. So for instance `cellElement.innerText = text;`still belongs here as the internal implementation of the table should be hidden for the consumer of the wrapper. It might be re-implemented in Flash for instance and the consumer of the API or of the wrapper should not notice any change. I'm open to discussing it (in person) but at the moment I think it's OK as it is.

Reply:
I have to change my mind on this one, @tomassatinsky has a point. Anyway it's now more like `agGridWrapper` than `agGridApiWrapper`.

Reply:
Then maybe everyone sees this wrapper as something else. I see it as thin api to unsafe backend of agGrid and you see it as wrapper to whole AgGrid. And you are correct we should discuss this with whole team. But if you merge this. There is no way back and nobody will refactor this, and it will become wrapper to whole aggrid

Reply:
So the question is whether we rename this file, or split this into two different files (and will it be beneficiary?).